nautobot/extras/api/nested_serializers.py

```diff
@@ -167,7 +175,7 @@ class Meta:
         ]
 
 
-class NestedJobResultSerializer(BaseModelSerializer):
+class NestedJobResultSerializer(WritableNestedSerializer):
```

Comment:
Should we allow writing/editing Nested Job Result Entries?

Reply:
See above comment. The only places we use NestedJobResultSerializer at present are read-only. As with JobLogEntry, `JobResult` doesn't support Notes or ObjectChanges at present so this change ends up not being *strictly* necessary but I think it's still desirable from a consistency standpoint.